exercises/pascals-triangle/instructions.md

```diff
@@ -0,0 +1,3 @@
+# Instructions
+
+Your task is to output the first N rows of Pascal's Triangle.
```

Comment:
Do we need more here? Some examples?

Reply:
An example explicitly showing the addition process like in https://en.wikipedia.org/wiki/Pascal%27s_triangle#/media/File:PascalTriangleAnimated2.gif might be handy. I think that's where students might get stuck. Once you've got over that, I don't think additional examples provide a lot of extra insight. The modeled triangle doesn't change, just the number of rows exported.